docs/reference/rgbd_hand_gesture_learner.md

```diff
@@ -0,0 +1,342 @@
+```python
+opendr.perception.multimodal_human_centric.rgbd_hand_gesture_learner.rgbd_hand_gesture_learner.get_hand_gesture_dataset(path, resolution)
+```
+
+This method is used to get the training and validation sets of the hand gesture dataset [[1]](#dataset) that are ready to be used with the `fit()` method.  
+If the dataset is not available under the given path, it will be downloaded and preprocessed.  
+
+**Parameters**:
+  
+- **path**: *str*   
+  Specifies the directory path where the dataset resides.  
+  If the dataset does not exist under the given path, it will be downloaded and saved under `path/hand_gestures`.    
+- **resolution**: *int, default=224*   
+  Specifies the resolution in which the images will be resized.  
+
+**Returns**:
+
+- **train_set**: *opendr.engine.datasets.DatasetIterator*   
+  The training set object that can be used with the `fit()` method to train the hand gesture classifier.   
+- **val_set**: *opendr.engine.datasets.DatasetIterator*   
+  The validation set object that can be used with the `fit()` method to train the hand gesture classifier.   
+- **n_class**: *int*  
+  The number of hand gestures in this dataset.  
+- **text_labels**: *list[str]*  
+  The list of textual labels of the hand gestures.   
+  If the prediction from the model is `k`, the textual description of hand gesture is `text_labels[k]`.   
+
+
+### Examples
+
+* **Training an RGBD hand gesture recognition model**.  
+  In this example, we will train a hand gesture classifier using the data provided by [[1]](#dataset).  
+  The training and validation data object can be constructed easily by using our convenient method `get_hand_gesture_dataset()` as follows:   
+
+  ```python
+  from opendr.perception.multimodal_human_centric.rgbd_hand_gesture_learner.rgbd_hand_gesture_learner import \
+    RgbdHandGestureLearner, get_hand_gesture_dataset 
+
+  # download the dataset to the current directory
+  train_set, val_set, n_class, text_labels = get_hand_gesture_dataset('.')
+  ```
+
+  Then, we can construct the classifier having the `densenet121` architecture with the pretrained ImageNet weights, and train the learner for 300 iterations as follows:  
+
+  ```python
+  learner = RgbdHandGestureLearner(n_class=n_class, architecture='densenet121', pretrained=True, iters=300)
+  performance = learner.fit(train_set, val_set)
+  ```
+
+* **Using pretrained hand gesture recognition model**.  
+  In this example, we will demonstrate how a pretrained hand gesture recognition model can be easily downloaded and used.   
+  At the moment, only a light-weight pretrained `mobilenet_v2` is provided so we will create a learner instance with `mobilenet_v2` as the architecture.  
+
+   ```python
+  from opendr.perception.multimodal_human_centric.rgbd_hand_gesture_learner.rgbd_hand_gesture_learner import \
+    RgbdHandGestureLearner 
```

Comment:
It would probably be better to not split the line so that it is easier to copy and paste the code
```suggestion
  from opendr.perception.multimodal_human_centric.rgbd_hand_gesture_learner.rgbd_hand_gesture_learner import RgbdHandGestureLearner 
```